indico/modules/rb_new/client/js/common/rooms/RoomDetailsModal.jsx

```diff
@@ -142,10 +142,18 @@ function RoomDetails({bookRoom, room, availability, attributes}) {
                         <Translate>Would you like to use this space?</Translate>
                     </Message>
                     <Segment attached="bottom">
-                        <Button color="green" onClick={() => bookRoom(room.id)}>
-                            <Icon name="check circle" />
-                            <Translate>Book it</Translate>
-                        </Button>
+                        {room.canUserBook && (
+                            <Button color="green" onClick={() => bookRoom(room.id, {isPrebooking: false})}>
+                                <Icon name="check circle" />
+                                <Translate>Start booking</Translate>
+                            </Button>
+                        )}
+                        {room.canUserPreBook && (
+                            <Button color="orange" onClick={() => bookRoom(room.id, {isPrebooking: true})}>
+                                <Icon name="check circle" />
```

Comment:
I believe we said we'd look for another icon?

Reply:
Anyway, it can be done after...

Reply:
That's true, I had forgotten. Any suggestions? We use the clock in some places to point out that a booking is actually a pre-booking but this makes sense as the pre-booking has been already created. I don't know if it'd be suitable in this case.

Reply:
In my opinion a clock is suitable also for creation. Since we use it in other places already, the users may recognize this icon as anything connected to prebooking